deep_qa/training/run_model.py

```diff
@@ -0,0 +1,117 @@
+
```

Comment:
And I might just put it at the top level: `deep_qa/run.py`.  And in `deep_qa/__init__.py`, you put the imports, so you can just do `from deep_qa import run_model, evaluate_model, load_model`.